OsmAnd/res/values/strings.xml

```diff
@@ -11,21 +11,21 @@
 	Thx - Hardy
 -->
 
-    <string name="switch_mode_by_tapping_on_widget">You can switch mode by taping on widget.</string>
-    <string name="map_widget_sunset_desc">Shows time of the next sunset or time left to sunset.</string>
-    <string name="map_widget_sunrise_desc">Shows time of the next sunrise or time left to sunrise.</string>
+    <string name="switch_mode_by_tapping_on_widget">Switch mode by tapping a widget.</string>
```

Comment:
```suggestion
    <string name="switch_mode_by_tapping_on_widget">Switch mode by tapping the widget.</string>
```
?

Reply:
Depends on the context, no?

Reply:
I don't know which one is correct, but "taping" is not it.